tests/flow_class_field/jsfmt.spec.js

```diff
@@ -1 +1 @@
-run_spec(__dirname, null, ["babylon"]);
+run_spec(__dirname, { parser: "babylon" });
```

Comment:
It's related to #1481 but now this code only parses in babylon and not flow anymore. Maybe the babylon fix should be reverted? cc @existentialism